package.json

```diff
@@ -32,8 +32,8 @@
     "migrate:save": "npm explore '@journaly/j-db-client' -- npm run migrate:save",
     "empty-db": "source .env && psql $DATABASE_URL  -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'",
     "reseed-db": "npm run empty-db && npm run migrate:up && npm run seed",
-    "translations:topics:pull": "ts-node -r tsconfig-paths/register -P tsconfig-base.json scripts/pull-topic-translations.ts",
-    "translations:topics:push": "ts-node -r tsconfig-paths/register -P tsconfig-base.json scripts/push-topic-translations.ts"
+    "translations:topics:pull": "ts-node -r tsconfig-paths/register -P tsconfig-base.json scripts/pull-topic-translations.mjs",
+    "translations:topics:push": "ts-node -r tsconfig-paths/register -P tsconfig-base.json scripts/push-topic-translations.mjs"
```

Comment:
Woah, first time seeing `.mjs` haha

Reply:
haha, yeah, it's kind of goofy. Guess it's a way to inform node that a file is a module and can use the es6 import syntax?